package.json

```diff
@@ -158,7 +158,7 @@
     "@ryansonshine/commitizen": "4.2.8",
     "@ryansonshine/cz-conventional-changelog": "3.3.4",
     "@schematics/angular": "18.2.12",
-    "@skyux/dev-infra-private": "github:blackbaud/skyux-dev-infra-private-builds#10.0.0-alpha.19",
+    "@skyux/dev-infra-private": "github:blackbaud/skyux-dev-infra-private-builds#10.0.0-alpha.20",
```

Comment:
Looks like `package-lock.json` did not pick up the change. Run this: ```
npm i -D @skyux/dev-infra-private@github:blackbaud/skyux-dev-infra-private-builds#10.0.0-alpha.20
```

Reply:
Good catch.